src/vm_memory_monitor.erl

```diff
@@ -447,3 +494,34 @@ read_proc_file(IoDevice, Acc) ->
         {ok, Res} -> read_proc_file(IoDevice, [Res | Acc]);
         eof       -> Acc
     end.
+
+
+%% Memory reported by erlang:memory(total) is not supposed to
+%% be equal to the total size of all pages mapped to the emulator,
+%% according to http://erlang.org/doc/man/erlang.html#memory-0
+%% erlang:memory(total) under-reports memory usage by around 20%
+-spec get_used_memory() -> Bytes :: integer().
+get_used_memory() ->
+    case application:get_env(rabbit, vm_memory_use_process_rss, false) of
```

Comment:
No because this module is moved to rabbitmq-common in the `resolve-deps-circles` branch to solve interdependencies between the broker, the Erlang client and the rabbitmq-common. But the configuration key will remain in the `rabbit` application.